MessageBird/Resources/MessageList.cs

```diff
@@ -0,0 +1,37 @@
+
+using System.Text;
+
+namespace MessageBird.Resources
+{
+    public class MessageLists : BaseLists<Objects.Message>
+    {
+        public MessageLists()
+            : base("messages", new Objects.MessageList())
+        {
+        }
+        
+        public override string QueryString
+        {
+            get
+            {
+                var baseList = (Objects.MessageList)Object;
+
+                var builder = new StringBuilder();
+
+                if (!string.IsNullOrEmpty(base.QueryString))
+                {
+                    builder.AppendFormat("{0}", base.QueryString);
+                }
+
+                // builder.AppendFormat("limit={0}", baseList.Limit);
+                // builder.AppendFormat("&");
+                // builder.AppendFormat("offset={0}", baseList.Offset);
```

Comment:
forgotten? I guess we need to append the limit and offset as well

Reply:
forgot to remove this... thank you! actually it is already appended in baseList..